website/docs/gator.md

```diff
@@ -299,6 +322,26 @@ gator expand --filename="manifest.yaml" –format="json"
 See `gator expand –help` for more details. `gator expand` will exit 1 if there
 is a problem parsing the configs or expanding the resources.
 
+## Bundling Policy into OCI Artifacts
+
+Policy can be bundled into OCI Artifacts for ingestion by `gator test|expand`.
+The bundle can be composed of any arbitrary file structure, which `gator` will
+walk recursively. Any files that do not end in `json|yaml|yml` will be ignored.
+`gator` does not enforce any file schema in the artifacts; it only requires that
+all files of the support extensions describe valid Kubernetes resources.
+
+We recommend using the [Oras CLI](https://oras.land/cli/) to create OCI
+artifacts. For example, to push a bundle containing the 2 local directories
+`constraints` and `template_library`:
+
```

Comment:
can we add use cases on why a user would choose to use OCI images vs yaml files?

Reply:
I am planning on following this up with a more detailed example/tutorial on how one might use this feature in a CI/CD workflow. I think the value add will be much stronger if we can publish some of our constraint libraries to a registry for people to use.

As a stop gap, I added some text above about how this might be useful in CI/CD.

Reply:
do we want to introduce this feature as alpha since gator is going to be beta in v3.11?

Reply:
Yea, good idea. I added a note in the docs (`gator.md`) that the image flag is in alpha.